patches/minecraft/net/minecraft/client/model/AgeableListModel.java.patch

```diff
@@ -0,0 +1,14 @@
+--- a/net/minecraft/client/model/AgeableListModel.java
++++ b/net/minecraft/client/model/AgeableListModel.java
+@@ -76,4 +_,11 @@
+    protected abstract Iterable<ModelPart> m_5607_();
+ 
+    protected abstract Iterable<ModelPart> m_5608_();
++
++   @Override
++   public void gatherAnimatedParts(java.util.function.BiConsumer<String, ModelPart> consumer) {
++      // This might be bad since changes in the iterables
```

Comment:
While true, the convention for these is the iterables to be an unchanging immutable list, so there should be no issue.

Reply:
Alright, will assume immutable.